### DISCUSSION TITLE
Is this possible?

### DISCUSSION THREAD

**Discussion by HithaishiDesai** [m4mallu/clonebot#75] [Q&A]
![IMG_20220806_104520](https://user-images.githubusercontent.com/78928904/183241011-360fdd35-331c-451c-b8f8-f975816adf1f.jpg)

Is it possible to make it just **CHANNEL**  because whenever I try to tap on chat Id sometimes i click on that 

THANKS 🙏🏻

**Comment by m4mallu** [comment]
Okay

**Comment by m4mallu** [comment]
Fixed : Fixed : https://github.com/m4mallu/clonebot/commit/d859540d378d4cf6ccb5c8f3a38b9b6295de8caf
